### ISSUE TITLE
I18n mvc

### ISSUE THREAD

**Issue by akleine01** [turquoiseowl/i18n#358]
I Triet several Times to geht it Run with ASP.net mvc.
I have looked to config etc. 
The Pot Files are already generated.
But vor eg the View will get translated.
Ist There a Sample mvc Project so i can look oder whats different.

**Comment by turquoiseowl** [comment]
No, sorry, there isn't an official sample. That is on the TODO list.

**Comment by sShrek1** [comment]
@akleine01 Create an repository with test project and link it here.
